### PR TITLE
logic of bookings index view page done

### PR DESCRIPTION
**Pull request by aquidam**
Logic working
Will need some CSS
Will also need to be refactored (the same block of code is repeated three times)

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -3,7 +3,10 @@ class BookingsController < ApplicationController
   before_action :set_animal, only: [:create]
 
   def index
-    @bookings = Booking.where("user_id = #{current_user}")
+    @bookings = Booking.where(user_id: current_user)
```
Comment (Pieter-JanCaals) [inline app/controllers/bookings_controller.rb]: Change `user_id: current_user` to `user: current_user`
This is cleaner code

File: app/views/bookings/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,60 @@
+<!-- Available instance variables :
```
Comment (Pieter-JanCaals) [inline app/views/bookings/index.html.erb:1]: Nice!

File: app/views/bookings/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,60 @@
...
 </div>
+<h1>My Bookings</h1>
+
+<!-- Checks if a user doesn't have any bookings -->
+<% if @bookings.size == 0 %>
+  <h3>You don't have any bookings yet!</h3>
+<% end %>
+
+<!-- Display the pending bookings -->
+<% if @pending.size > 0 %>
```
Comment (Pieter-JanCaals) [inline app/views/bookings/index.html.erb]: Replace `@pending.size > 0` by `@pending.size.positive?`
Playing Rubocop here :)

File: app/views/bookings/index.html.erb [hunk 4]
```diff
@@ -1,6 +1,60 @@
...
+    <%= link_to "Edit", animal_path(booking.animal) %>
+    <%= link_to "Cancel", booking_path(booking.id),
+        method: :delete,
+        data: { confirm: "Are you sure you want to delete this booking?" } %>
+    <br>
+  <% end %>
+<% end %>
+
+<!-- Display the confirmed bookings -->
+<% if @confirmed.size > 0 %>
```
Comment (Pieter-JanCaals) [inline app/views/bookings/index.html.erb]: Same

File: app/views/bookings/index.html.erb [hunk 5]
```diff
@@ -1,6 +1,60 @@
...
+    <span>Animal: <%= booking.animal.name %></span>
+    <span>Start_date: <%= booking.start_date %></span>
+    <span>End_Date: <%= booking.end_date %></span>
+    <p>Content: <%= booking.content %></p>
+    <br>
+  <% end %>
+<% end %>
+
+<!-- Display the rejected bookings -->
+<% if @rejected.size > 0 %>
```
Comment (Pieter-JanCaals) [inline app/views/bookings/index.html.erb]: Same

**Review by Jerokas — APPROVED** [review]

File: app/views/bookings/index.html.erb [hunk 6]
```diff
@@ -1,6 +1,60 @@
...
+<!-- Display the pending bookings -->
+<% if @pending.size > 0 %>
+  <h3>Pending bookings:</h3>
+  <% @pending.each do |booking| %>
+    <span>Animal: <%= booking.animal.name %></span>
+    <span>Start_date: <%= booking.start_date %></span>
+    <span>End_Date: <%= booking.end_date %></span>
+    <p>Content: <%= booking.content %></p>
+    <%= link_to "Edit", animal_path(booking.animal) %>
+    <%= link_to "Cancel", booking_path(booking.id),
```
Comment (Pieter-JanCaals) [inline app/views/bookings/index.html.erb]: change `booking.id` by `booking`

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (aquidam) [reply app/controllers/bookings_controller.rb]: Done!

File: app/views/bookings/index.html.erb (hunk 3 above)
Comment (aquidam) [reply app/views/bookings/index.html.erb]: Done! By the same token, on line 17 I changed @bookings.size == 0 to @bookings.size.zero?

File: app/views/bookings/index.html.erb (hunk 4 above)
Comment (aquidam) [reply app/views/bookings/index.html.erb]: Idem

File: app/views/bookings/index.html.erb (hunk 5 above)
Comment (aquidam) [reply app/views/bookings/index.html.erb]: Idem

File: app/views/bookings/index.html.erb (hunk 6 above)
Comment (aquidam) [reply app/views/bookings/index.html.erb]: Done!

**Pull request merged**